common/options/options_test.go

```diff
@@ -1091,3 +1092,135 @@ func TestDeprecationWarning(t *testing.T) {
 		})
 	})
 }
+
+func TestPasswordPrompt(t *testing.T) {
+	testtype.SkipUnlessTestType(t, testtype.UnitTestType)
+
+	pw := "some-password"
+	expectPrompt := "Enter password for mongo user:\n"
```

Comment:
Testing on a specific, human-readable string will cause a spurious failure if we ever reword this phrase. For example,  (e.g., to have the actual username rather than `mongo user`). The test would be more resilient if it checked just for, e.g., `password.*:\n`

Reply:
Good call, done.